apps/web/src/pages/Profile/AccountSettings/RecruitmentAvailability.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+
+import { PoolCandidateStatus, User } from "@gc-digital-talent/graphql";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { Link, Well } from "@gc-digital-talent/ui";
+
+import ApplicationCard from "~/pages/ProfileAndApplicationsPage/components/TrackApplications/ApplicationCard";
```

Comment:
If a component is used on more than one page we should move it to `components`. Importing stuff from different pages causes issues with code-splitting.

Reply:
Fixed by [c443ad2](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9971/commits/c443ad2a7aafac7c4447e60be2baeaf3c7f64730)